input/fsh/canonical/COVIDInitialHealthAssessmentQuestionnaire.fsh

```diff
@@ -0,0 +1,348 @@
+Alias: $usage-context-type = http://terminology.hl7.org/CodeSystem/usage-context-type
+Alias: $sct = http://snomed.info/sct
+Instance: COVIDInitialHealthAssessmentQuestionnaire
+InstanceOf: Questionnaire
+Usage: #definition
+
+* identifier[0].use = #official
+* identifier[=].value = "COVIDInitialHealthAssessmentQuestionnaire"
+* identifier[=].period.start = "2023-07-19"
+* identifier[+].use = #temp
+* identifier[=].value = "QuestionTemplate-CitC-COVID19-InitialAssessment"
+* identifier[=].period.end = "2023-07-19"
+
+* version = "0.1.5"
+* name = "COVIDInitialHealthAssessmentQuestionnaire"
+* title = "COVID19 Initial Health Assessment Questionnaire"
```

Comment:
This is different to what is currently seeded to the server, which is correct?

Reply:
The IG version, name and title shall prevail.  Also the seed resource should be renamed COVIDInitial**Health**AssessmentQuestionnaire.json if we are going to keep using it.